listenbrainz/webserver/static/js/src/user/UserFeedback.tsx

```diff
@@ -522,11 +517,15 @@ export default class UserFeedback extends React.Component<
                         dataTarget="#PinRecordingModal"
                       />,
                     ];
+                    const recording_msid =
+                      listen.track_metadata.additional_info?.recording_msid;
+                    const recording_mbid =
+                      listen.track_metadata.additional_info?.recording_mbid;
```

Comment:
We have utilities to do this which can support the various formats we use, and various places where the fields might be: 
I've been trying to use it everywhere: https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/static/js/src/utils/utils.tsx#L133-L138

Reply:
fixed.